components/LearnCourse/LearnCoursePages.vue

```diff
@@ -30,17 +30,12 @@
       </div>
       <div v-if="activeCourse" class="learn-course-pages__content__preview">
         <UiLink :url="activeCourse.url">
-          <!-- TODO: investigate why this particular img is not being populated -->
-          <!-- <nuxt-img
+          <nuxt-img
```

Comment:
This isn't working for me @eddybrando 

- checked branch preview, no image
- ran `npm install`, `npm run generate`, and `nuxt start` locally to mimic the production environment, and no images

please let me know if I'm doing it incorrectly.. .
![Screenshot 2023-07-28 at 7 49 54 AM (2)](https://github.com/Qiskit/qiskit.org/assets/6276074/dabb8b38-02ab-4f48-b193-443d86539ec1)